app/src/main/java/com/keylesspalace/tusky/ComposeActivity.java

```diff
@@ -1880,6 +1939,10 @@ public Intent build(Context context) {
             if (replyingStatusAuthor != null) {
                 intent.putExtra(REPLYING_STATUS_AUTHOR_USERNAME_EXTRA, replyingStatusAuthor);
             }
+            if (jsonMediaAttachments != null) {
+                intent.putExtra(JSON_MEDIA_ATTACHMENTS_EXTRA, jsonMediaAttachments);
```

Comment:
I would rather use `intent.putParcelableArrayListExtra(` and put in the list of attachments. Shorter and probably faster than de/serializing with Gson.